services/storage/virtualharddisk/virtualharddisk.go

```diff
@@ -82,6 +87,8 @@ func getVirtualHardDisk(c *wssdcloudstorage.VirtualHardDisk, group string) *stor
 			HyperVGeneration:    c.HyperVGeneration,
 			DiskFileFormat:      c.DiskFileFormat,
 			ContainerName:       &c.ContainerName,
+			SourcePath:          &c.SourcePath,
+			SourceType:          c.SourceType,
```

Comment:
I think we should remove this. Otherwise the cloudagent is exposing the vhd's download url.

Reply:
Good Point. Removed.